modules/ROOT/pages/auto-updates.adoc

```diff
@@ -7,6 +7,38 @@ By default, the OS performs continuous auto-updates via two components:
  * https://github.com/coreos/rpm-ostree[rpm-ostree] handles multiple on-disk OSTree deployments, and it can switch between them at boot-time.
  * https://github.com/coreos/zincati[Zincati] continuously checks for OS updates and applies them via rpm-ostree.
 
+== Configuring Update/Rollout Wariness
+
+The Zincati service runs on a Fedora CoreOS node and checks with the remote Cincinnati service to see when updates are available. As part of the request it can send along a configuration value known as `rollout_wariness` (see https://github.com/coreos/zincati/blob/master/docs/usage/auto-updates.md#phased-rollouts-client-wariness-canaries[documentation]) to let the server know how eager, or how risk-averse the node is to receiving updates. `rollout_wariness` can be a floating point value between `0.0` (most eager) and `1.0` (most conservative). To configure a node to receive an update as soon as it is available you can set the `rollout_wariness` to `0.0` in a configuration file within `/etc/zincati/config.d/` using https://github.com/coreos/fcct[FCCT]:
```

Comment:
`It can be configured with a custom "rollout wariness" value to let the server...`

Reply:
+1